src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -427,3 +427,191 @@
     type: command
     short-summary: Stop a web app.
 """
+
+helps['functionapp'] = """
+    type: group
+    short-summary: Manage your function app.
+"""
+
+helps['functionapp create'] = """
+    type: command
+    short-summary: Create a function app.
+    examples:
+        - name: Create a basic function app.  Name must be unique to yield a unique FQDN;
+                for example, MyUniqueApp.azurewebsites.net.
+          text: >
+            az functionapp create
+            -g MyResourceGroup
+            -p MyPlan
+            -n MyUniqueApp
+            -s MyStorageAccount
```

Comment:
Does this actually print on multiple lines?  If so, why is this not a single line?